test/read-line.l

```diff
@@ -2,18 +2,34 @@
 
 (init-unit-test)
 
+#|
 (defun test-read-line
   (max-char)
   (length (read-line
            (make-string-input-stream
             (apply #'concatenate string
                    (make-list max-char :initial-element "a")))))
   )
+|#
+
+(defun test-read-line
+  (max-char)
+  (let ((sm (make-string-output-stream)))
```

Comment:
@YoheiKakiuchi why we have to redefine this?

Reply:
It is for very long string.
using concatenate or make-line may fail for making long string more than stack size...